spec/catcher_spec.rb

```diff
@@ -16,21 +16,28 @@
     let(:options) { { :id => id, :locale => locale } }
     let(:cache_key) { 'example-en-1' }
     let(:resource) { "http://example.com/en/1" }
-    let(:request) { stub(response:'{"example":{"id":1}}') }
+    let(:request) { stub(response: response, body: response) }
```

Comment:
If we use `double` then we get `undefined method `double' for #<RSpec::ExampleGroups::Catcher...>`, which can be remedied by removing the line `config.mock_framework = :mocha` from the `spec_helper.rb` file. The downside of this approach is that we then have to fix all the tests for Catcher making this a much larger pull request.

I'd rather keep my footprint as low as possible for now, and provide a subsequent pull request in the future to upgrade all of the tests to our preferred syntax.